src/libraries/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -159,14 +160,21 @@ public void Start(int backlog)
             _active = true;
         }
 
-        // Closes the network connection.
+        /// <summary>
+        /// Closes the network connection.
+        /// </summary>
```

Comment:
Triple-slash docs are only required for new API-s, `Stop()` [already has docs](https://learn.microsoft.com/en-us/dotnet/api/system.net.sockets.tcplistener.stop?view=net-7.0), with a different, more meaningful text.